torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -1602,6 +1603,24 @@ const std::vector<uint64_t>& ProcessGroupNCCL::groupRanks() const {
   return options_->global_ranks_in_group;
 }
 
+void ProcessGroupNCCL::addEphemeralTimeout(
```

Comment:
nit: update PR desc to mention ephemeral and include more notes about how it works and what its for

Reply:
Sure, I added more details to PR desc.